lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin`.chomp
```

Comment:
This will print `fatal: No such remote 'origin'` to stderr when `origin` is undefined. Should be something like this instead:

```ruby
 repo = `git remote get-url origin 2>/dev/null`.chomp